tango_simlib/simdd_json_parser.py

```diff
@@ -44,14 +44,10 @@ def __init__(self):
         self._device_override_class = {}
 
     def parse(self, simdd_json_file):
-        """Read simulator description data from json file
+        '''Read simulator description data from json file.
```

Comment:
Looks like this was changed accidentally.  Triple-quoted docstrings must use double quote characters for [pep8](https://www.python.org/dev/peps/pep-0008/#string-quotes).